tools/bioimaging/bioimage_inference.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="bioimage_inference" name="Load model from BioImage and make inferences" version="0.0.5">
```

Comment:
```suggestion
<tool id="bioimage_inference" name="Process image using model from BioImage Model Zoo" version="0.0.5">
```
Does a typical user of this tool know what "inference" is? Just a suggestion, maybe something less technical?

Reply:
thank you @kostrykin for your suggestions